description.md

```diff
@@ -38,6 +38,73 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Assignments
 
+Assignments are unassisted practical activities that learners undertake individually or in teams over a longer timespan.
+The primary objective of an assignment is to create a compelling scenario that allows learners to apply the concepts they have learned.
+
+#### Storing Assignments
+
+The central component of an assignment is the assignment statement, serving the purpose of offering a detailed explanation for each task.
+Additionally, it may offer an introduction, details about the evaluation process, and further referrences.
+Typically, this content is stored in either markdown format or as a PDF, allowing for the incorporation of media files.
+
+An assignment might contain any of the following:
+
+- Utility Files
+
+  These files can be exemplars or code snippets meant to facilitate solution development.
+  They often come into play when addressing topics tangential to the main assignment task.
+  These files are designed for utility purposes and should remain unaltered by learners; they are not subject to evaluation.
+
+- Skeleton Files
+
+  These serve as the starting point of the assignment and establish a similar pattern, which is crucial for automated testing.
+  Depending on the subject matter, skeleton files range from templates and fill-in-the-blank outlines to simple guidelines.
+
+- Automated Evaluation Program
+
+  Such a program (a checker) is generally a script that evaluates a given solution and compares the result to a series of expected metrics.
+  Optionally, the checker might use pairs of inputs and expected results to evaluate the solution on a set of predefined conditions.
```

Comment:
Why isn't the statement not part of this bulleted list?

Reply:
Also, I think that this list should be made more Computer Science agnostic. As it is, it looks like this is useful only if you a CS educator.